src/applications/toe/config/form.js

```diff
@@ -122,10 +123,12 @@ const formConfig = {
   // submitUrl: '/v0/api',
   submit: () =>
     Promise.resolve({ attributes: { confirmationNumber: '123123123' } }),
+  // ToDo: Will need to uncomment below line & comment line above once sumit url is available
+  // transformForSubmit: transformTOEForm,
```

Comment:
You can set a temporary submitUrl, even if it doesn't work, to make it easier to test the transformation.